fastlane/Fastfile

```diff
@@ -246,23 +248,23 @@ end
       abort_on_violations: false
     )
 
-    UI.message("Checking release notes strings translation status...")
+    UI.message('Checking release notes strings translation status...')
     check_translation_progress(
       glotpress_url: 'https://translate.wordpress.com/projects/simplenote/ios/release-notes/',
       abort_on_violations: false
     )
 
     ios_update_metadata(options)
-    sanitize_appstore_keywords()
+    sanitize_appstore_keywords
     ios_lint_localizations(input_dir: 'Simplenote', allow_retry: true)
-    ios_bump_version_beta()
+    ios_bump_version_beta
 
     # Wrap up
-    version = ios_get_app_version()
-    removebranchprotection(repository:GHHELPER_REPO, branch: "release/#{version}")
-    setfrozentag(repository:GHHELPER_REPO, milestone: version, freeze: false)
-    create_new_milestone(repository:GHHELPER_REPO)
-    close_milestone(repository:GHHELPER_REPO, milestone: version)
+    version = ios_get_app_version
+    removebranchprotection(repository: GHHELPER_REPO, branch: "release/#{version}")
+    setfrozentag(repository: GHHELPER_REPO, milestone: version, freeze: false)
+    create_new_milestone(repository: GHHELPER_REPO)
+    close_milestone(repository: GHHELPER_REPO, milestone: version)
```

Comment:
💄 : Not really related to the goal of that PR, but now that we got rid of the mentions of `GHHELPER_*` in release-toolkit 6.0.0 and the "term" `GHHELPER` is kinda legacy in our tooling at that point, I think it could be nice to rename this constant in your `Fastfile` to `GITHUB_REPO` rather than `GHHELPER_REPO` 🤷

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/6bde715607693c183e325d091a005d06043691f6